src/RedisPrefixTrait.php

```diff
@@ -94,4 +94,44 @@ protected function getPrefix() {
     return $this->prefix;
   }
 
+  /**
+   * From the given timestamp, with arbitrary increment as decimal, get
+   * the decimal value
+   *
+   * @param int|string $timestamp
+   *   "TIMESTAMP[.INCREMENT]" string
+   *
+   * @return string
+   *   "TIMESTAMP.INCREMENT" string.
+   */
+  public function getNextIncrement($timestamp = null) {
```

Comment:
This is magic! I'm sorry the implementation fo this function is not ideal, but believe me I keep profiling and even thought it could probably be written shorter, it doesn't have any significative impact on the execution.